library/core/src/mem/manually_drop.rs

```diff
@@ -56,8 +56,7 @@ use crate::ptr;
 /// working with [pinned] data, where reusing the memory without calling the destructor could lead
 /// to Undefined Behaviour.
 ///
-/// [`mem::zeroed`]: fn.zeroed.html
-/// [`MaybeUninit<T>`]: union.MaybeUninit.html
+/// [`mem::zeroed`]: MaybeUninit::<T>::zeroed
```

Comment:
Rustdoc will not resolve this `<T>`: https://github.com/rust-lang/rust/issues/62834. But also I think it was meant to link to `mem::zeroed`, not `MaybeUnint::zeroed`, so you can leave it out altogether and it will still be resolved.
```suggestion
///  [`MaybeUninit<T>`]: MaybeUninit
```

Reply:
Thank you for your suggestion. 

`///  [`MaybeUninit<T>`]: MaybeUninit` shows `unresolved link error`...

Reply:
I fixed it with `[`MaybeUninit<T>`]: crate::mem::MaybeUninit`